gitlab/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,238 @@
+name: Gitlab
+files:
+- name: gitlab.yaml
+  options:
+  - template: instances
+    options:
+    - name: gitlab_url
+      required: true
+      description: |
+        Master URL to probe for service health status
+        If you are using gitlab CE and not EE, use this url with
+        the authorization token: http://localhost/?token=<TOKEN>
+      example: <GITLAB_URL>
```

Comment:
```suggestion
      value:
        type: string
```

Reply:
Updated! That fixed the weird spacing :)